main.cc

```diff
@@ -867,12 +867,6 @@ int main(int ac, char** av) {
 
             distributed_loader::ensure_system_table_directories(db).get();
 
-            static sharded<cdc::cdc_service> cdc;
```

Comment:
@elcallio was there any particular reason you put the initialization of cdc service in this place or is it fine to move it?